### PR TITLE
DASH-992 Governance pages fixes

### PR DESCRIPTION
**Pull request by trukhilio**

### CODE REVIEW

File: client/src/hooks/governance.ts [hunk 1]
```diff
@@ -224,90 +227,126 @@ export const useOveriewWallets = (): {
...
+              }
+            }
+          } catch (error) {
+            log.error(
+              `Error fetching data for wallet ${walletPublicKey}:`,
+              error,
+            );
+          }
+        }),
+      );
```
Comment (andreyvEze) [inline client/src/hooks/governance.ts:294]: I think we can move this fetching logic out of component

File: client/src/pages/GovernancePage/components/WalletVotesDetailsModal/components/MyVoteDetails/MyVoteDetails.tsx [hunk 2]
```diff
@@ -35,44 +27,29 @@ export const MyVoteDetails: FC<Props> = ({
...
-          }) || 0}{' '}
-          FIO
-        </span>
-      </p>
-      {lastUpdated && (
+      <div className={classes.detailsPowerContainer}>
         <p className={classes.detailsItem}>
           <span className={classes.detailsLabel}>
-            Board Voting Power Last Updated:
+            Current {powerLabelSuffix ? <>Board </> : ''}Voting Power:
```
Comment (andreyvEze) [inline client/src/pages/GovernancePage/components/WalletVotesDetailsModal/components/MyVoteDetails/MyVoteDetails.tsx:33]: why we need to do this way? `<>Board </>`

**Review by andreyvEze — APPROVED** [review]
check commments

File: client/src/pages/GovernancePage/components/WalletVotesDetailsModal/components/MyVoteDetails/MyVoteDetails.tsx (hunk 2 above)
Comment (trukhilio) [reply client/src/pages/GovernancePage/components/WalletVotesDetailsModal/components/MyVoteDetails/MyVoteDetails.tsx:33]: Looks like it uses in multiple places

File: client/src/hooks/governance.ts (hunk 1 above)
Comment (trukhilio) [reply client/src/hooks/governance.ts:294]: Ok

**Pull request merged**